content/geometry/introduction.adoc

```diff
@@ -9,18 +9,18 @@ endif::[]
 
 == Motivation
 
-A 3D asset is a digital entity that includes a 3D geometric representation of an object along with its attributes and metadata. 
+A 3D asset (or **3D Data File**) is a digital entity that includes a 3D geometric representation of an object along with its attributes and metadata. 
```

Comment:
In my opinion, a 3D asset is the whole package including the asset file. A 3D data file to me is just the glTF (or FBX, USD) file. So the metadata is not part of the 3D data file.